packages/nala/build.sh

```diff
@@ -2,72 +2,31 @@ TERMUX_PKG_HOMEPAGE=https://gitlab.com/volian/nala
 TERMUX_PKG_DESCRIPTION="Commandline frontend for the apt package manager"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.7.2
+TERMUX_PKG_VERSION=0.11.1
 TERMUX_PKG_SRCURL=https://gitlab.com/volian/nala/-/archive/v${TERMUX_PKG_VERSION}/nala-v${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=0ac394ab2bd924c28c513a8e8bfc9e899a21f2f8fa5e5d77366cb20e0be8676e
+TERMUX_PKG_SHA256=4ce87d75e785e45b1638817077797a3fcd4a15059615455fc1af0127c70cf5cd
 TERMUX_PKG_DEPENDS="python-apt"
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
 _PYTHON_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python/build.sh; echo $_MAJOR_VERSION)
 
-TERMUX_PKG_RM_AFTER_INSTALL="
-lib/python${_PYTHON_VERSION}/site-packages/__pycache__
```

Comment:
Please re-add this so the `__pycache__` folder is removed from the package

Reply:
I'm pretty sure here is no need, since that folder is not being generated. Python packages build with `pyproject.toml` ensure build isolation by defauild, following PEP 518.

Reply:
Right, the pycache folder is in the nala subdirectory, so path to remove should be `lib/python${_PYTHON_VERSION}/site-packages/nala/__pycache__`:

```
$ dpkg -c output/nala_0.11.1_all.deb | grep pycache
drwx------ builder/builder     0 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/
-rw------- builder/builder 10037 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/cache.cpython-310.pyc
-rw------- builder/builder 16291 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/utils.cpython-310.pyc
-rw------- builder/builder 25206 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/install.cpython-310.pyc
-rw------- builder/builder 16919 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/error.cpython-310.pyc
-rw------- builder/builder  4465 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/constants.cpython-310.pyc
-rw------- builder/builder  1724 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/__init__.cpython-310.pyc
-rw------- builder/builder  9085 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/summary.cpython-310.pyc
-rw------- builder/builder  4388 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/search.cpython-310.pyc
-rw------- builder/builder  6939 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/debfile.cpython-310.pyc
-rw------- builder/builder 14295 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/nala.cpython-310.pyc
-rw------- builder/builder 21085 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/fetch.cpython-310.pyc
-rw------- builder/builder 13693 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/options.cpython-310.pyc
-rw------- builder/builder 18419 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/downloader.cpython-310.pyc
-rw------- builder/builder  6214 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/rich.cpython-310.pyc
-rw------- builder/builder 14657 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/history.cpython-310.pyc
-rw------- builder/builder  1053 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/__main__.cpython-310.pyc
-rw------- builder/builder 29691 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/dpkg.cpython-310.pyc
-rw------- builder/builder  9344 2022-10-23 09:13 ./data/data/com.termux/files/usr/lib/python3.10/site-packages/nala/__pycache__/show.cpython-310.pyc
```

Reply:
Ok, I fixed the removal path